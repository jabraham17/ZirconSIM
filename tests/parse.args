-control '8 + 9'
