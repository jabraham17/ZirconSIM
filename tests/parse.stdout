{NUM, "8"}
{PLUS, ""}
{NUM, "9"}
success
